neonvm-runner/neonvm-guest/vminit-TODO

```diff
@@ -0,0 +1,20 @@
+## FIXME: This is unused with neonvm-guest. It's here just to remind that some
+## of this functionality still needs to be moved to systemd unit files
```

Comment:
Alternatively, we can do those from neonvm-daemon before neonvm-payload is started.